bottles/frontend/windows/gamescope.py

```diff
@@ -32,9 +32,10 @@ class GamescopeDialog(Adw.Window):
     switch_scaling = Gtk.Template.Child()
     toggle_borderless = Gtk.Template.Child()
     toggle_fullscreen = Gtk.Template.Child()
+    switch_fsr = Gtk.Template.Child()
+    spin_sharpening_strength = Gtk.Template.Child()
     btn_save = Gtk.Template.Child()
     btn_cancel = Gtk.Template.Child()
```

Comment:
We shouldn't modify 'unrelated' lines for `git blame`.

```suggestion
    btn_cancel = Gtk.Template.Child()

```